lib/linux_admin/yum.rb

```diff
@@ -87,8 +87,6 @@ def self.repo_list(scope = "enabled")
       parse_repo_list_output(output)
     end
 
-    private
-
```

Comment:
We should have private_class_method on all of the below methods to compensate for the intent.

Reply:
That breaks the current specs, so I couldn't tell if they were actually meant to be private.